src/page-modules/contact/ticket-control-and-fee/index.tsx

```diff
@@ -22,10 +31,78 @@ export const TicketControlAndFeeContent = (
   };
 
   return (
-    <section>
-      <form onSubmit={onSubmit}>
-        <button type="submit">Send</button>
-      </form>
-    </section>
+    <div>
+      <SectionCard title={PageText.Contact.ticketControl.title}>
+        <FormSelector state={state} send={send} />
+      </SectionCard>
+      {state.matches('agreement1') && (
+        <FeeComplaintForm state={state} send={send} onSubmit={onSubmit} />
+      )}
+      {state.matches('refundForm') && (
+        <div>
+          {/* Todo : Create and add <RefundForm/> */}
+          refund
+        </div>
+      )}
+      {state.matches('feedbackForm') && (
+        <div>
+          {/* Todo : Create and add <FeedbackForm/> */}
+          feedback
+        </div>
+      )}
+    </div>
+  );
+};
+
+type FormSelectorProps = {
+  state: any;
+  send: (event: any) => void;
+};
+
+const FormSelector = ({ state, send }: FormSelectorProps) => {
```

Comment:
Should we add the selected form as a query parameter? Then we could link directly to the `feeComplaintForm` or `refundForm` etc